cmd/httpclient/main.go

```diff
@@ -15,12 +15,12 @@
 //
 // Examples:
 //
-//   ./httpclient -dns-transport system ...
-//   ./httpclient -dns-transport godns ...
-//   ./httpclient -dns-transport doh ...
-//   ./httpclient -dns-transport dot ...
-//   ./httpclient -dns-transport tcp ...
-//   ./httpclient -dns-transport udp [-dns-udp-server <addr>:<port>] ...
+//   ./httpclient -dns-server system:// ...
+//   ./httpclient -dns-server godns:// ...
+//   ./httpclient -dns-server https://<url> ...
+//   ./httpclient -dns-server ./httpclient -dns-server dot://<domain>|<addr>:<port> ...
```

Comment:
> `<domain>|<addr>:<port>`

https://developers.cloudflare.com/1.1.1.1/dns-over-tls/

Reply:
nit: I believe there is a duplicate `./httpclient` above?

Reply:
With respect to `<addr>:<port>`, could you check whether it works? (I believe cloudflare to be `1.1.1.1:853` in this case. My initial take was that we needed a domain to validate, but it may be that the cert also says that it is valid for `1.1.1.1`. Since you say the docs say that an IP address is also okay, I'd be curious to see whether it also works using an IP address.

Reply:
Two solutions:
- Change here
- Change `dnsclient`.

I think the latter is the right thing to do. (Should I?)

Reply:
Go for it!